app/controllers/hyrax/homepage_controller.rb

```diff
@@ -63,6 +63,8 @@ def index
       end
     end
 
+    def browserconfig; end
```

Comment:
What is this for?

Reply:
Is it related to this file? `app/views/hyrax/homepage/browserconfig.xml.erb`

Reply:
This is the controller action for that file, it's how you set the favicon on windows browsers